xbdd-web/src/__tests__/tag-list/components/TagListFilterButtonsView-test.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { shallow, mount } from "enzyme";
+import TagListFilterButtonsView from "../../../modules/tag-list/components/TagListFilterButtonsView";
+
+const filterStates = {
+    passedSelected: true,
+    undefinedSelected: true,
+    failedSelected: true,
+    skippedSelected: true,
+};
+
+test("TagListButtonsView renders", () => {
```

Comment:
Probably best to wrap it in a describe block having 'TagListButtonsView'  and then it blocks with 'renders ....' ?

Reply:
Done